engine/src/main/scala/cromwell/engine/db/DataAccess.scala

```diff
@@ -649,4 +650,14 @@ trait DataAccess extends AutoCloseable {
       }
     }
   }
+
+  def refreshWorkflowMetadataSummaries(startMetadataId: Long, startMetadataTimestamp: Option[OffsetDateTime])
+                                      (implicit ec: ExecutionContext): Future[Long] = {
+    self.refreshMetadataSummaries(startMetadataId, startMetadataTimestamp)
+  }
+
+  def getWorkflowStatus(id: WorkflowId)
+                       (implicit ec: ExecutionContext): Future[Option[WorkflowState]] = {
+    self.getStatus(id) map { _ map WorkflowState.fromString }
```

Comment:
I created #935 that has the previously mentioned change in it.

The `Semigroup` code embedded inside the `refreshMetadataSummaries` transaction is too complicated for me to ramp up on at the moment. So I'm just going to leave the scalaz code in there alone and give that a LGTM. 😉